### PR TITLE
Docs: fix typos in documentation

### PR DESCRIPTION
**Pull request by xyb**
----
📚 Documentation preview 📚: https://cpython-previews--118815.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/bz2.rst [hunk 1]
```diff
@@ -324,7 +324,7 @@ Using :func:`compress` and :func:`decompress` to demonstrate round-trip compress
     ... Donec rhoncus quis sapien sit amet molestie. Fusce scelerisque vel augue
     ... nec ullamcorper. Nam rutrum pretium placerat. Aliquam vel tristique lorem,
     ... sit amet cursus ante. In interdum laoreet mi, sit amet ultrices purus
-    ... pulvinar a. Nam gravida euismod magna, non varius justo tincidunt feugiat.
+    ... pulvinar a. Nam gravida euismod magna, non various justo tincidunt feugiat.
```
Comment (vsajip) [inline Doc/library/bz2.rst]: This is not a typo, it's Latin - you can leave it as is.

File: Doc/library/bz2.rst [hunk 2]
```diff
@@ -364,7 +364,7 @@ Writing and reading a bzip2-compressed file in binary mode:
...
```
Comment (vsajip) [inline Doc/library/bz2.rst]: This is not a typo, it's Latin - you can leave it as is.

**Review by vsajip — CHANGES_REQUESTED** [review]
The other changes look fine to me.

File: Doc/library/bz2.rst (hunk 2 above)
Comment (Privat33r-dev) [inline Doc/library/bz2.rst]:
```suggestion
    ... pulvinar a. Nam gravida euismod magna, non varius justo tincidunt feugiat.
```

File: Doc/library/bz2.rst (hunk 1 above)
Comment (Privat33r-dev) [inline Doc/library/bz2.rst]:
```suggestion
    ... pulvinar a. Nam gravida euismod magna, non varius justo tincidunt feugiat.
```

**Review by Privat33r-dev — CHANGES_REQUESTED** [review]
Please, revert the latin changes as suggested. Other changes are good, thank you for your contribution.

**Comment by xyb** [comment]
@vsajip @Privat33r-dev Thank you for your advices. I have made the requested changes; please review again.

**Review by Privat33r-dev — APPROVED** [review]

**Review by vsajip — APPROVED** [review]

**Pull request merged**